app/src/main/java/com/edu/sun/oereminder/ui/report/ReportContract.kt

```diff
@@ -0,0 +1,14 @@
+package com.edu.sun.oereminder.ui.report
+
+import com.edu.sun.oereminder.ui.base.BasePresenter
+import com.edu.sun.oereminder.ui.base.BaseView
+
+interface ReportContract {
+    interface View : BaseView {
+
+    }
```

Comment:
```suggestion
    interface View : BaseView {
    }
```
t thấy nhiều file ô cách r nhiều file lại cách thêm 1 dòng như v. bỏ đi cho file n ngắn bớt